_posts/2021-05-11-unstructured-knowledge-sharing.markdown

```diff
@@ -0,0 +1,105 @@
+---
+layout: epic
+title: "Unstructured Knowledge Sharing"
+date: 2021-05-11
+categories: [teams, culture, people]
+author: steve-hicks
+---
+
+Something something why is knowledge sharing important? blah blah blah we emphasize it at Artsy.
+
+<!-- more -->
+
+We have a handful of regularly scheduled meetings in place at Artsy devoted to knowledge sharing:
+
+- Team-based knowledge share meetings
+- Open office hours for anyone to join when they want help with something
+- Lunch & learns for presenting across all of Artsy engineering
+- Show & tell, where attendees bring recent learnings or explorations to demo to the group
+- Peer learning groups that are dedicated to learning a specific topic in development
+
+As [Ash][ash] mentioned in [his article on knowledge shares][knowledge-shares], these are all great _structured_
+opportunities for us to share knowledge. Learning is easy to de-prioritize in the face of shipping features, and
+scheduling these meetings reinforces learning as a priority.
+
+But what about the _unstructured_ ways in which we share knowledge? Structured sharing time demonstrates that the
+team is interested in spreading knowledge; _unstructured_ sharing time demonstrates that the spreading of knowledge
+has become _the default_ mode for the team. Instead of the team forming habits of working in isolation or hoarding
+expertise, they've formed habits of learning from and teaching each other.
+
+This article describes a few things we do that are unstructured and unscheduled, all of which demonstrate our
+emphasis on learning at Artsy.
+
+## The #dev-help Slack Channel
+
+<blockquote class="twitter-tweet"><p lang="en" dir="ltr">Shortest blog post of all time:<br>“Add a <a href="https://twitter.com/hashtag/dev?src=hash&amp;ref_src=twsrc%5Etfw">#dev</a>-help channel to your slack where devs can pose questions to the wider team. You won’t regret it!”<br>-- Chris Pappas</p>&mdash; Artsy Open Source (@ArtsyOpenSource) <a href="https://twitter.com/ArtsyOpenSource/status/1357819867638812672?ref_src=twsrc%5Etfw">February 5, 2021</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+We have many dev-specific channels set up in the Artsy Slack, but one in particular has become a massively
+important knowledge-base for the entire team: the #dev-help channel.
+
+This channel is a place for engineers to post when they're stuck. All engineers watch the #dev-help channel (to
+some degree) and offer support when they can. Each issue is threaded to avoid noise in the channel. When the
+question is answered, it gets marked as "solved" by applying a ✅ reaction. [Pavlos][pavlos] set up a Slack app
+that auto-applies the ✅ reaction when someone says "solved" in the thread.
+
+![A question asked and answered in our #dev-help slack](/images/2021-05-11-unstructured-knowledge-sharing/dev-help.png)
```

Comment:
@ashleyjelks @dzucconi do I have permission from both of you to use [a screenshot](https://github.com/artsy/artsy.github.io/blob/89ae9de06d09a9cbcbd1d2c924ced59cd223b42c/images/2021-05-11-unstructured-knowledge-sharing/dev-help.png) of [this example from #dev-help about palette icons](https://artsy.slack.com/archives/CP9P4KR35/p1619549433137300)?

I like it because it's short and to the point, and it shows the magic of dev-help. I'm happy to scrub your avatar and/or name from the conversation if you'd like -- just let me know.  

![image](https://user-images.githubusercontent.com/1627089/116927444-49fabf00-ac21-11eb-83a8-96cd0d399512.png)

Reply:
Fine by me